commands/build_test.go

```diff
@@ -0,0 +1,71 @@
+package commands
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+// nolint:deadcode,unused
+func setupTest(tb testing.TB) func(tb testing.TB) {
+	return func(tb testing.TB) {
+		os.Unsetenv("BUILDX_GIT_LABELS")
+	}
+}
+
+func TestAddGitProvenanceDataWithoutEnv(t *testing.T) {
+	labels, err := addGitProvenance(nil, ".", "")
+	if err != nil {
+		t.Error("No error expected")
+	}
+	if labels != nil {
+		t.Error("No labels expected")
+	}
+}
+
+func TestAddGitProvenanceDataWithoutLabels(t *testing.T) {
+	os.Setenv("BUILDX_GIT_LABELS", "full")
+	labels, err := addGitProvenance(nil, ".", "")
```

Comment:
What is the "." directory here? Test should create its own testing repo and not expect to be run at a specific location(or am I missing something?).

Reply:
Creating my own test git repo now.